opencti-platform/opencti-front/src/private/components/workspaces/dashboards/Dashboard.jsx

```diff
@@ -799,13 +802,24 @@ const DashboardComponent = ({ workspace, noToolbar }) => {
       }}
     >
       {!noToolbar && (
-        <WorkspaceHeader
-          handleAddWidget={handleAddWidget}
-          workspace={workspace}
-          config={manifest.config}
-          handleDateChange={handleDateChange}
-          variant="dashboard"
-        />
+        <>
+          <WorkspaceHeader
+            handleAddWidget={handleAddWidget}
+            workspace={workspace}
+            variant="dashboard"
+            config={manifest.config}
+            handleDateChange={handleDateChange}
+          />
+          {isFABReplaced && (
+            <div style={{ marginTop: 8 }}>
+              <DashboardTimeFilters
```

Comment:
I refactored the time filters, extracting the code from WorkspaceHeader and using it just here, in the dashboard.